### PR TITLE
Added linear_search.cpp

### PR DESCRIPTION
**Pull request by ashiscs**

### CODE REVIEW

**Comment by ashiscs** [comment]
@Bhavnaharitsa please review?

File: Algorithms/Searching/linear_search.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	int i; 
+	for (i = 0; i < n; i++) 
+		if (arr[i] == x) 
+			return i; 
+	return -1; 
+} 
+
+int main(void) 
+{ 
+	int arr[] = { 2, 3, 4, 10, 40 };
```
Comment (Bhavnaharitsa) [inline Algorithms/Searching/linear_search.cpp:17]: make it dynamic not static

File: Algorithms/Searching/linear_search.cpp [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	for (i = 0; i < n; i++) 
+		if (arr[i] == x) 
+			return i; 
+	return -1; 
+} 
+
+int main(void) 
+{ 
+	int arr[] = { 2, 3, 4, 10, 40 }; 
+	int x = 10;
```
Comment (Bhavnaharitsa) [inline Algorithms/Searching/linear_search.cpp:18]: no single letter variables

File: Algorithms/Searching/linear_search.cpp [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+		if (arr[i] == x) 
+			return i; 
+	return -1; 
+} 
+
+int main(void) 
+{ 
+	int arr[] = { 2, 3, 4, 10, 40 }; 
+	int x = 10; 
+	int n = sizeof(arr) / sizeof(arr[0]);
```
Comment (Bhavnaharitsa) [inline Algorithms/Searching/linear_search.cpp:19]: same

**Comment by parjanyaacoder** [comment]
Can i work on this issue

**Comment by Bhavnaharitsa** [comment]
I think @ashiscs is already working 
Please wait for him to make the changes 
You can take up other issues

**Comment by ashiscs** [comment]
@parjanyaacoder I will resolve this as early as possible.

**Pull request closed without merging**